install.sh

```diff
@@ -22,7 +22,7 @@ function set_variable_from_command() {
         result=$(bash -c "${!DEFAULT_COMMAND}") || {
```

Comment:
```suggestion
        result=$(bash -c "${DEFAULT_COMMAND}") || {
```

Given the modification below, this should no longer an indirect dereference (the test script needs also to be updated)